stable/etcd-operator/values.yaml

```diff
@@ -154,4 +154,4 @@ etcdCluster:
     ## Ref: https://kubernetes.io/docs/user-guide/node-selection/
     nodeSelector: {}
     securityContext: {}
-    tolerations: {}
+    tolerations: []
```

Comment:
Extracted this fix from regression introduced in https://github.com/helm/charts/pull/16085 into a dedicated PR: https://github.com/helm/charts/pull/17068